### DISCUSSION TITLE
How to save a property which value comes from a join() collection?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#893] [Q&A] [answered]
Hi

When I have a property which value (trought mv-value, or inside innerHTML) comes from a collection, that I join()
-> I'm unable to save this property data

Do I need to replace some characters inside value, to be able to save it? Or is this a bug?

My codepen [here](https://codepen.io/galinhalx/pen/PoRmgdp):

**Comment by GalinhaLX** [comment, marked as answer]
Solved 

Need to use **mv-default**
`mv-default="[join(myList.name,' ')]"`

Codepen is also corrected to reflect answer

**Reply by LeaVerou** [reply]
Note that this would also make it editable. If you don't want that, you can specify `mv-storage="auto"` and it will be saved :)

**Reply by karger** [reply]
I've used `mv-storage="inherit"` to force saving of mv-values; what does `auto` do and how does it differ?

**Reply by LeaVerou** [reply]
Just a typo, it should be inherit.
